src/backend/tcop/postgres.c

```diff
@@ -133,6 +135,7 @@ extern char *optarg;
 extern char *savedSeqServerHost;
 extern int savedSeqServerPort;
 
+struct curl_context_t curl_context;
```

Comment:
The name of this global variable "curl_context", seems ambiguous, code readers can't know what is this curl context used for. maybe "curl_context_rps" or "curl_context_ranger" is better, indicates this curl context is used for Ranger Plugin Service.